frontend/src/components/personal/preferencePage/PreferencesPage.scss

```diff
@@ -24,12 +24,21 @@ body {
     display: flex;
     flex-direction: row;
     margin-bottom: 5%;
+
+    @media (max-width: 600px) {
```

Comment:
I think you can write a mixin (https://sass-lang.com/documentation/at-rules/mixin) and use it, instead of writing it multiple times.
Also, we can have a variable for `600px`